src/Raven.Server/Rachis/Follower.cs

```diff
@@ -203,13 +207,22 @@ private void FollowerSteadyState()
                         return;
                     }
                     _debugRecorder.Record("Processing entries is completed");
-                    _connection.Send(context, new AppendEntriesResponse
+                    var curAer = new AppendEntriesResponse { CurrentTerm = _term, LastLogIndex = lastAcknowledgedIndex, LastCommitIndex = lastCommit, Success = true };
+
+                    bool shouldLog = false;
+                    if (sw.Elapsed.TotalMilliseconds > 1000)
```

Comment:
why do we want to log it so often @karmeli87 ?

Reply:
enlarge the interval to 10 seconds